pt_PT.json

```diff
@@ -0,0 +1,138 @@
+  "Save All": "Guardar Todos",
+  "Close": "Fechar",
+  "Close All": "Fechar Todos",
+  "Quit": "Sair",
+  "Edit": "Editar",
+  "Undo": "Desfazer",
+  "Redo": "Refazer",
+  "Cut": "Cortar",
+  "Copy": "Copiar",
+  "Paste": "Colar",
+  "Select All": "Seleccionar Tudo",
+  "Search": "Procurar",
+  "Find...": "Encontrar...",
+  "Find Next": "Encontrar Próximo",
+  "View": "Vista",
+  "Debug": "Depuração",
+  "Run DST": "Correr DST",
+  "Tools": "Ferramentas",
+  "Steam": "Steam",
+  "Workshop": "Workshop",
+  "Server": "Servidor",
+  "Start": "Começar",
+  "Stop": "Parar",
+  "Restart": "Recomeçar",
+  "Configuration": "Configuração",
+  "Settings": "Definições",
+  "Help": "Ajuda",
+  "Forum": "Fórum",
+  "About": "Acerca",
+  "Details": "Detalhes",
+  "Deinstall": "Desinstalar",
+  "Authentication": "Autenticação",
+  "Login with Steam": "Ligar-se com a Steam",
+  "If you want to upload Mods on Steam-Workshop, you need to be authenticated by Steam.": "Se quiser fazer upload de Mods para a Steam-Workshop, precisa autenticar-se pela Steam.",
+  "This program is licensed under OpenSource.": "Este progrmaa está sob uma licensa OpenSource.",
+  "Version:": "Versão:",
+  "Electron:": "Electron:",
+  "Render Engine:": "Mecanismo the Renderização:",
+  "CPU:": "CPU:",
+  "Network Interfaces:": "Interfaces de Rede:",
+  "Operating System": "Sistema Operativo",
+  "Platform:": "Platforma:",
+  "Release:": "Lançamento:",
+  "Architecture:": "Arquitectura:",
+  "Endianness:": "Extremidade:", //according to Wikipedia
+  "Hostname:": "Hostname:", //we don't translate this
+  "Memory (RAM):": "Memoria (RAM):",
+  "This file is an Core file! If you modify this file, your Game can be crash.": "Este ficheiro é um ficheiro Core. Se o modificares, o teu jogo pode terminar inesperadamente.",
+  "Mod-Editor": "Mod-Editor",
+  "Source": "Fonte", //I don't believe this is the best option...
+  "API-Version": "Versão-API",
+  "Name": "Nome",
+  "Description": "Descrição",
+  "Author": "Autor",
+  "Version": "Versão",
+  "Priority": "Prioridade",
+  "Compatiblity": "Compatibilidade",
+  "Reign of Giants": "Reign of Giants",
+  "Icon": "Ícone",
+  "Clients required": "Exigido para Clients",
+  "Clients MOD": "MOD para Clientes",
+  "Tags": "Etiquetas",
+  "This Section is currently under development.": "Esta secção está em desenvolvimento.",
+  "No Items Available": "Sem items",
+  "Page": "Página",
+  "Items": "Items",
+  "Unknown": "Desconhecido",
+  "Dialog": "Diálogo",
+  "Message": "Mensagem",
+  "Projects": "Projectos",
+  "Core": "Core",
+  "Preview": "Pré-visualizar",
+  "Properties": "Propriedades",
+  "State": "Estado",
+  "Rember...": "Rember...",
+  "Save the changes before closing?": "Guardar alterações antes de Sair?",
+  "Cancel": "Cancelar",
+  "General": "Geral",
+  "Shortcuts": "Atalhos",
+  "Languages": "Línguas",
+  "Interval:": "Intervalo:",
+  "<u>W</u>orkspace:": "(<u>W</u>) Área de Trabalho:", //I believe the underline is for shortcut purposes
```

Comment:
You can use another character, not only `W`; Please update it for your language specific character. If you dont know, which is a "default", check eclipse IDE or other for a sample.